v1.0/docs/api-reference/operations.md

```diff
@@ -1,5 +1,6 @@
 ---
 layout: docwithnav
+title: docs.26570/docs/api-reference/operations.md
```

Comment:
Maybe you can take the filename but strip off the prefix up to and include "docs/" ?  This is going to be different every run.